acme/setup.py

```diff
@@ -58,10 +58,10 @@
 
 jax_requirements = [
     'chex',
-    'jax==0.3.6',  # Update when TF2.9 is release.
-    'jaxlib==0.3.15',  # Update when TF2.9 is release.
-    'dm-haiku',
-    'flax',
+    'jax==0.4.1',  # Update when TF2.9 is release.
+    'jaxlib==0.4.1',  # Update when TF2.9 is release.
+    'dm-haiku==0.0.10',
+    #'flax',
```

Comment:
@varun-k-s why did you have to comment out this package?

Reply:
I got an "error: subprocess-exited-with-error" when including flax as a package in acme/setup.py.
flax was already being installed as a requirement for some other packages, so commenting the line out did not cause an issue.

Reply:
And assuming everything still worked fine @varun-k-s right?

Reply:
Yes everything works fine

Reply:
Amazing! If you could test that it all works on main now that it has been merged that would be a huge help as well to just confirm all your hard work is officially complete and working for folks to use :) Then I can ping folks who have been interested in using your work!

Reply:
@varun-k-s